packages/react-dom/src/shared/ReactDOMUnknownPropertyHook.js

```diff
@@ -21,9 +21,13 @@ import {
 import isCustomComponent from './isCustomComponent';
 import possibleStandardNames from './possibleStandardNames';
 
+let ReactDebugCurrentFrame = null;
+
 let validateProperty = () => {};
 
 if (__DEV__) {
+  ReactDebugCurrentFrame = ReactSharedInternals.ReactDebugCurrentFrame;
```

Comment:
It doesn't exist outside of DEV, which is true in other "hook" files too. Other "hook" files don't have DEV blocks (they just assume they're called only in DEV) so I didn't put the condition there. Since this file already had a condition, I put it conditionally.